tools/validators/ontology_validator/yamlformat/tests/entity_type_manager_test.py

```diff
@@ -249,5 +250,67 @@ def testAnalyzeHandlesNamespaces(self):
     self.assertTrue(parent.HasFindingTypes(
         [findings_lib.DuplicateExpandedFieldSetsWarning]))
 
+  def testGetCompleteFieldSetsOI(self):
```

Comment:
I think to complete the tests, you need one more with inheritance between types.

type0 {field1, field2}
type1 inherits from type0
type1 {field11}
type2 {field1, field11}

  manager.Analyze()
    function_output = manager.GetCompleteFieldSetsOI()
    
    test_output = field1, field11, type1, type2


what do you think?

Reply:
@charbull I had a more extensive test previously, but writing out the test_output by hand was painful. I will add inheritance.